src/utils/Logger.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export class Logger {
+    static instance: Logger;
+
+    static getInstance() {
+        if (!this.instance) {
+            this.instance = new Logger();
+        }
+        return this.instance;
+    }
+    info(tag: string, message: string, context?: any) {
```

Comment:
I don't like the way you are passing tag and message arguments. This is just too much effort. Why not just pass variadic arguments if we want to create our own class. And why even create a custom class if it is the same as console. That adds another line of imports basically to every file. We don't need to import console object because it is global. That task doesn't make sense to me. @Veryyapeee let me know

Reply:
Shit, I just realised that @T0ffik didn't add a condition based on env which should enable/disable logs.
That's basically the reason why we have to use global class, to disable logs in production.

The arguments makes much sense, it would make logs standardized which is very important.
Logs are important, and managing them with standardized way is big +

Reply:
And also just make all the methods static. We don't need an instance of class and all of this singleton pattern. This class doesn't hold any state so everything should be static just for ease of use.

Reply:
+1 for that